docs/Makefile

```diff
@@ -1,20 +0,0 @@
-# Minimal makefile for Sphinx documentation
```

Comment:
I'd just put `Makefile` and `make.bat` removals in a separate PR.

Reply:
Done with https://github.com/ansible/ansible-sdk/pull/41